strawberryfields/gbs/similarity.py

```diff
@@ -32,9 +55,53 @@
     orbits
     orbit_cardinality
     event_cardinality
+
+Creating a feature vector
+-------------------------
+
+A feature vector of a graph can be created by choosing a collection of orbits or events and
+evaluating their probabilities with respect to GBS sampling with the embedded graph. These
+probabilities are then selected to be elements of the feature vector. Evaluating the
+probabilities of orbits or events can be achieved through two approaches:
+
+- **Direct sampling:** infer the probability of orbits or events from a set of sample data.
+
+- **Monte Carlo (MC) approximation:** generate samples within a given orbit or event and use them
+  to approximate the probability.
+
+This module provides functions to evaluate the probability of orbits and events through MC
+approximation. For an orbit or event :math:`E`, several samples from :math:`E` are drawn
+uniformly at random using :func:`orbit_to_sample` or :func:`event_to_sample`. Suppose :math:`N`
+samples :math:`\{S_{1}, S_{2}, \ldots , S_{N}\}` are generated. For each sample, this function
+calculates the probability :math:`p(S_i)` of observing that sample from a GBS device programmed
+according to the input graph and mean photon number. The sum of the probabilities is then
+rescaled according to the cardinality :math:`|E|` and the total number of samples:
+
+.. math::
+    p(E) \approx \frac{1}{N}\sum_{i=1}^N p(S_i) |E|,
+
+The sample mean of this sum is an estimate of the rescaled probability :math:`p(E)`.
+
+MC estimation is available through the following functions:
+
+.. autosummary::
     prob_orbit_mc
     prob_event_mc
+
+One canonical method of constructing a feature vector is to pick event probabilities
+:math:`p_{E_{k, n_{\max}}}` with all events :math:`E_{k, n_{\max}}` having a maximum photon count
+:math:`n_{\max}` in each mode. If :math:`\mathbf{k}` is the vector of selected events,
+the resultant feature vector is
+
+.. math::
+    f_{\mathbf{k}} = (p_{k_{1}}, p_{k_{2}}, \ldots)

```

Comment:
I think `p_{k_{i}}` is not yet defined? We should have something like `p_{k} := p_{E_{k, n_{\max}}}`, right?

Reply:
Good spot! Have fixed this